project/src/main/java/controller/HomeController.java

```diff
@@ -1,27 +1,40 @@
 package controller;
 
+import javax.annotation.PostConstruct;
 import javax.inject.Inject;
 
 import br.com.caelum.vraptor.Controller;
 import br.com.caelum.vraptor.Path;
+import br.com.caelum.vraptor.Post;
 import br.com.caelum.vraptor.Result;
 import dao.TesteDAO;
 import factory.DAOFactory;
 import model.Teste;
+import model.User;
 
 @Controller
 public class HomeController {
 
     @Inject
     private Result result;
+    
+    /*public HomeController(Result result) {
+		this.result = result;
+	}*/

```

Comment:
Remova codigo comentado.

Reply:
Construtor nao e necessario porque o CDI (Context and Dependency Injection) se encarrega de criar uma instancia do objeto Result (por isso existe o @Inject ali em cima da variavel).